src/cf-forms/src/molecules/form-fields.less

```diff
@@ -21,7 +21,7 @@
                 grid-row-start: 1;
                 grid-row-end: 3;
 
-                border: 1px solid @input-border;
+                border: 1px solid @form-field-border;
```

Comment:
This is the gray border for the box and circle, are we sure we want to call it `form-field-border`?

Reply:
I'm not sure this is even right for the large target inputs. Form fields wrap lots of elements, including the labels paired with text inputs and select elements. Since these really are borders for the input label, not the form-field, they should probably stay as "input-#"

Reply:
That's okay, but we need something that is only used for the radio buttons/checkboxes in `form-fields.less`, and isn't shared across `select.less` and `text-input.less`. Otherwise it's too easy to be working on one type of input and inadvertently change a color on another. What name do you think?

Reply:
These pseudo elements are a faux input, created because we couldn't reliably style the radio and checkbox input types (are we certain that's still the case without worrying about IE7, 8, or 9???). They should continue to use the standard input border variables because they use the same colors as a text input correct? 

For the large target labels, how about `large-target-border`, that way they're never highjacked for anything else.

Reply:
`@large-target-widgets-border`?

Reply:
I feel like we want to have some connection between the CSS class and the variable it uses. I named these `@form-field-*` because they are used within `m-form-field`, like how `@input-*` is used in `a-text-input` and `@select-*` is used in `.a-select`. My point above is that if we have variables stretched across different classes it makes updates brittle, and it may be an anti-pattern (pending something like ye olde visual regression testing), so if they aren't used in another class, why not name them after the class within which they're used? I changed `@input-lg-target-bg` to `@form-field-lg-target-bg` under this premise because it is used in `m-form-field`, not `a-text-input` (also, anything with large in the name should probably follow this existing format and order, so `@large` should not come first).

Reply:
Could something like `@form-field-input-border` work to add a bit more context? Otherwise, I think this is fine and shouldn't prevent merging.

Reply:
@virginiacc Good idea. I've updated it to that. We may revisit the naming overall for forms, but out of scope for this PR.